deployment/deploy-dev.md

```diff
@@ -47,27 +39,19 @@ Go to the `deployment` folder and run `pwsh ./deploy-az-dev-bootstrapper.ps1 -Ap
 
 Subsequent deployments can be run as follows.
```

Comment:
I loved this change when you did that initially, so much easier to iterate over changes in a dev loop.